kolibri/plugins/learn/assets/src/modules/topicsRoot/handlers.js

```diff
@@ -15,7 +15,11 @@ export function showChannels(store) {
       }
       const channelRootIds = channels.map(channel => channel.root);
       ContentNodeResource.fetchCollection({
-        getParams: { ids: channelRootIds, user_kind: store.getters.getUserKind },
+        getParams: {
+          ids: channelRootIds,
+          include_coach_content:
+            store.getters.isAdmin || store.getters.isCoach || store.getters.isSuperuser,
```

Comment:
This policy of "include coach content only if user is at least a coach" can be moved to the backend, especially if there is no use case for the user to explicitly set `include_coach_content = false` through the client.

Reply:
We exclude the content node endpoints (aside from resume and next steps) from doing any session checking to allow for indefinite caching of the requests, if we remove this GET param and defer to the backend for determining the user role, we are reintroducing the same issue that this PR is trying to resolve.